src/app/base/services/project/project-apollo.service.ts

```diff
@@ -1430,10 +1430,10 @@ export class ProjectApolloService {
         },
         fetchPolicy: 'no-cache'
       });
-      const vc = query.valueChanges.pipe(
-        map((result) => result['data']['getGatesIntegrationData'])
-      );
-      return [query, vc];
+    const vc = query.valueChanges.pipe(
+      map((result) => result['data']['getGatesIntegrationData'])
```

Comment:
this has

Reply:
i not sure what our common approach is (i think without get) -> but this will include a backend change